concepticondata/conceptrelations.tsv

```diff
@@ -446,3 +446,6 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 2678	HAIR OR FUR	narrower	580	FUR
 2678	HAIR OR FUR	narrower	1040	HAIR
 2381	FEATHER OR FUR OR HAIR	narrower	2678	HAIR OR FUR
+2672	UNCLE OR AUNT	narrower	1272	AUNT
```

Comment:
sorry for bothering: we have the silent, no-where-stated convention to sort the "a or b" concepts alphabetically. This makes searching easier.